google/ggrc-core
# GGRC-4225 Comparison window doesn't show Notes for Risk

# Issue description

Comparison window and "info-pin" don't show Notes for Risk

# Steps to test the changes

1. Create a Risk as admin
2. Log as Global Reader and Propose changes for Notes
3. As admin user open comparison window and look at Notes
_Actual Result_: Comparison window doesn't show Notes for Risk
_Expected Result_: Comparison window should show Notes for Risk

# Solution description

"**Info-page**" uses "_base_objects/info.mustache_" template for "Risk", but "**Info-pin**" uses "_risks/info.mustache_".

"_risks/info.mustache_" is equal to "_base_objects/info.mustache_" except "Note" field.

I decided to remove unneeded ("_risks/info.mustache_") template.

# Sanity checklist




# PR Review checklist
